src/main/java/sign_up/ui/SignUpView.java

```diff
@@ -0,0 +1,111 @@
+package sign_up.ui;
+
+import sign_up.interface_adapters.SignUpController;
+import sign_up.interface_adapters.SignUpUserInputData;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+
+
+public class SignUpView extends JPanel implements ActionListener {
+    private final SignUpController signUpController;
```

Comment:
What Is this Class? I don't see it in this commit. 
Also isn't SignUpUserInputData acting as the Controller?

Reply:
This is the panel for taking user input when signing up. Its analagous to the login view. if you look at the sign up view test, you can see what i mean.
Also, I think I just forgot to push my file for the controller. you should be able to see it now.